modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/MarketplaceStorePortlet.java

```diff
@@ -500,18 +505,15 @@ protected void processPortletParameterMap(
 			"supportsHotDeploy", new String[] {Boolean.TRUE.toString()});
 	}
 
-	@Override
-	@Reference(unbind = "-")
-	protected void setOAuthManager(OAuthManager oAuthManager) {
-		super.setOAuthManager(oAuthManager);
-	}
-
 	@Reference
 	protected AppLocalService appLocalService;
 
 	@Reference
 	protected AppService appService;
 
+	@Reference
+	protected OAuthManager oAuthManager;
```

Comment:
@slnn Please make this change ins separate commit to highlight the reason you need a "protected" here

Reply:
I have separate the commit to highlight the reason that needs a "protected"  in the new update